app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ */
+
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "comments";
+@import "likes";
+@import "photos";
+@import "users";
+@import "custom";
+
+
+a {
+  color: red;
+  text-decoration: none;
+} 
+.navbar-btn {
+  display:inline-block;
+  margin-left:10px;
+  float: right;
+}
+
+.upload-btn {
+  display:inline-block;
+  margin-left:10px;
+  float: left;
+}
+
+header .title a{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space